tests/Doctrine/Tests/DBAL/Functional/StatementTest.php

```diff
@@ -373,7 +374,9 @@ public function testExecWithRedundantParameters() : void
      */
     public function testFetchColumnNonExistingIndex(int $index) : void
     {
-        if ($this->connection->getWrappedConnection() instanceof PDOConnection) {
+        $wrappedConnection = $this->connection->getWrappedConnection();
+
+        if ($wrappedConnection instanceof PDOConnection || $wrappedConnection instanceof PDOSqlsrvConnection) {
```

Comment:
Why isn't `PDOSqlsrvConnection` sub-type of `PDOConnection`?

Reply:
I did some changes and now Connection and Statement in PDOSqlsrv are subtypes of specific doctrine PDO classes. Previously I wanted it to be similarly to the Sqlsrv implementation, but it is also possible to just extends PDO classes and do some overrides. Now it seems to be simpler.